bigtable/tests/unit/test_client.py

```diff
@@ -27,34 +27,32 @@ def _invoke_client_factory(self, client_class):
         return _create_gapic_client(client_class)
 
     def test_without_emulator(self):
-        from google.cloud.bigtable.client import _CLIENT_INFO
-
         client_class = mock.Mock()
         credentials = _make_credentials()
         client = _Client(credentials)
+        client_info = client._client_info = mock.Mock()
 
         result = self._invoke_client_factory(client_class)(client)
 
         self.assertIs(result, client_class.return_value)
         client_class.assert_called_once_with(
-            credentials=client._credentials, client_info=_CLIENT_INFO
+            credentials=client._credentials, client_info=client_info
         )
 
     def test_with_emulator(self):
-        from google.cloud.bigtable.client import _CLIENT_INFO
-
         client_class = mock.Mock()
         emulator_host = emulator_channel = object()
         credentials = _make_credentials()
         client = _Client(
             credentials, emulator_host=emulator_host, emulator_channel=emulator_channel
         )
+        client_info = client._client_info = mock.Mock()
```

Comment:
Ditto. We can use `mock.create_autospec` with `instance=True` to make doubly sure we don't use any properties on `ClientInfo` that don't exist.